1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,21 @@
-function setAlarm() {}
+function setAlarm() {
+  let alarmSetValue = document.getElementById("alarmSet").value;
+  let timeDisplay = document.getElementById("timeRemaining");
```

Comment:
worth looking into the **Prettier** code formatter for vscode, it can help to tidy up your files on save :)